generated/nidcpower/session.py

```diff
@@ -21,7 +21,23 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._session.abort()
 
 
-class _SessionBase(object):
+class _Channel(object):
+    def __init__(self, vi, library, encoding):
+        self._vi = vi
+        self._library = library
+        self._encoding = encoding
+
+    def __getitem__(self, repeated_capability):
+        '''Set/get properties or call methods with a repeated capability (i.e. channels)'''
+        if isinstance(repeated_capability, list):
+            rep_cap_list = [str(r) if str(r).lower().startswith('') else '' + str(r) for r in repeated_capability]
+        else:
+            rep_cap_list = [str(repeated_capability) if str(repeated_capability).lower().startswith('') else '' + str(repeated_capability)]
+
+        return _RepeatedCapbilities(vi=self._vi, repeated_capability=','.join(rep_cap_list), library=self._library, encoding=self._encoding, freeze_it=True)
+
+
+class _RepeatedCapbilities(object):
     '''Base class for all NI-DCPower sessions.'''
```

Comment:
Seems odd that a class named _RepeatedCapbilities is 'Base class for all NI-DCPower sessions.'

Reply:
I haven't come up with a perfect name for it but _SessionBase is probably better than _RepeatedCapabilities.

When I read _RepeatedCapabilities, I think of a collection of repeated capabilities such as "0" or "ScriptTrigger2".